File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

int solution(int a, int b, int c) {
    int answer = 0;
    if (a!=b && a!=c && b!=c) {
            answer = a+b+c;
        } else if((a==b && a!=c) || (a==c && a!=b) || (b==c && b!=a)){
            answer = (a+b+c) * (int)(pow(a,2) + pow(b,2) + pow(c,2));
        } else {
            answer = (a+b+c) * (int)(pow(a,2) + pow(b,2) + pow(c,2)) * (int)(pow(a,3) + pow(b,3) + pow(c,3));
        }
        return answer;
    return answer;
}

File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    if a != b and a != c and b != c:
        answer = (a+b+c)
    elif (a == b and a != c) or (b == c and b != a) or (c == a and c != b):
        answer = ((a+b+c) * (a**2 + b**2 + c**2))
    else:
        answer = ((a+b+c) * (a**2 + b**2 + c**2) * (a**3 + b**3 + c**3))

    return answer